### DISCUSSION TITLE
Gold Panning

### DISCUSSION THREAD

**Discussion by SaxW** [EveronLife/EveronLife#100] [Ideas]
Had an idea that rather than mining for gold, you could pan for gold - ie
![171979050-d8d9f11d-238e-46be-bf1f-3c11dcf1a12c.png](https://user-images.githubusercontent.com/53883603/172271177-697b7913-bd92-4de7-817a-f86868676529.png)

There are a few rivers on the map that fit the bill.
![171979434-2be3d433-f388-49fa-a772-c1817af5ec7d.png](https://user-images.githubusercontent.com/53883603/172271188-fd1fb751-9243-4984-a191-f6ac99188708.png)
![171979453-56dc9817-e34a-428e-8305-f9d346dcc1bb.png](https://user-images.githubusercontent.com/53883603/172271198-60fe74ff-d5fd-4bc9-836a-651a3ac32454.png)

**Comment by borknbeans** [comment]
There is the possibility of having both. Gold panning could be a low level gold mining option that has less requirements but less reward, while gold mining could be a riskier later game option.

**Comment by jasper-one** [comment]
I love the idea of having a varied experience for the different gathers, particularly late-game ones like potentially gold. It would suck to just have every mining type the exact same. Big fan of anything we can do to have multiple gameplay elements. 

The rivers are also awesome, great to have the variation from just going to a quarry/pile of rocks.
